src/client/components/app/VaultDetail/VaultDetailPanels.tsx

```diff
@@ -351,7 +351,7 @@ export const VaultDetailPanels = ({
             <StyledCardHeader header={vaultNameTitle} />
             {displayAddToken ? (
               <RelativeContainer onClick={handleAddToken}>
-                <StyledImg src={MetamaskLogo} />
+                <StyledImg src={MetamaskLogo} alt="Add token to Metamask" />
```

Comment:
nit: Should this `alt` be i18n instead?

Reply:
I would not invest the time to translate aria-labels to be honest.
I think 100% we should not support assistive screen readers in yearn.finance main webpage. 
To that, add the translate (vietnamese for example). It is a crazy amount of overhead and lots of things could end up wrong for somebody using them (and more if they translate the site).

Reply:
Fair point 👍